src/composables/accounts.ts

```diff
@@ -8,13 +8,43 @@ import type {
   INetwork,
 } from '../types';
 import { tg } from '../store/plugins/languages';
-import { FAUCET_URL, buildSimplexLink, getAccountNameToDisplay } from '../popup/utils';
+import {
+  FAUCET_URL,
+  PROTOCOL_AETERNITY,
+  PROTOCOL_BITCOIN,
+  buildSimplexLink,
+  getAccountNameToDisplay,
+} from '../popup/utils';
 import { AeScan } from '../lib/AeScan';
+import { testAccount } from '../popup/utils/testsConfig';
 
 export function useAccounts({ store }: IDefaultComposableOptions) {
   // TODO in the future the state of the accounts should be stored in this composable
   const activeIdx = computed((): number => store.state.accounts?.activeIdx || 0);
-  const accounts = computed((): IAccount[] => store.getters.accounts || []);
+  const accounts = computed((): IAccount[] => {
+    const baseAccounts: IAccount[] = (store.getters.accounts || []).map((acc: IAccount) => ({
+      ...acc,
+      protocol: PROTOCOL_AETERNITY,
```

Comment:
Instead of hardcoding the protocol type in the Account object, we can derive the Protocol type from the account address. We can introduce a method `identifyAccountProtocol` in this PR or in my per as per this comment https://github.com/aeternity/superhero-wallet/pull/2193/files#r1274653420

Reply:
Yes, we can derive it from the address. But in my opinion it's more readable and easier to debug to have the `protocol` property. Imagine you want to `console.log` the account data - you'll immediately know which protocol the account is.

Also as this property is not a thing that would change in time it's better to set it once instead of calculating it whenever we want to establish the protocol.

Reply:
My proposal is to calculate it once and then store it as part of `IAccount` interface.